test_runner/fixtures/neon_fixtures.py

```diff
@@ -1564,8 +1635,12 @@ def http_client(self, auth_token: Optional[str] = None) -> PageserverHttpClient:
             is_testing_enabled_or_skip=self.is_testing_enabled_or_skip,
         )
 
+    @property
+    def workdir(self) -> Path:
+        return Path(os.path.join(self.env.repo_dir, f"pageserver_{self.id}"))
```

Comment:
I think `repo_dir` is a `Path`

```suggestion
        return self.env.repo_dir / f"pageserver_{self.id}"
```

Reply:
You're right, I'll put a little commit in my next PR that touches this file, to pick it up (the next PR is the one that actually uses multiple pageservers for real)